app/api/business/seller_edit_business.py

```diff
@@ -168,8 +151,8 @@ def get_agreement_status(supplier, user_info):
         'canUserSign': can_user_sign_agreement,
         'signed': signed_agreement,
         'startDate': start_date.strftime('%Y-%m-%d'),
-        'currentAgreement': agreement,
-        'newAgreement': new_agreement
+        'currentAgreement': agreement.serialize() if agreement else {},
```

Comment:
this would be better if it is None.
```suggestion
        'currentAgreement': agreement.serialize() if agreement else None,
```